styles.css

```diff
@@ -389,6 +384,74 @@ textarea {
   color: #172b4d;
 }
 
+.mobile {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 900px;
+  background-color: #6070ff;
+  transform: translateY(-200%);
+  list-style: none;
+  padding-top: 100px;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  transition: transform 0.5s;
+  opacity: 0.9;
+}
+
+.showMenu {
+  transform: translateY(0);
+}
+
+.hamburger {
+  position: fixed;
+  top: 3%;
+  margin-right: 24px;
+  z-index: 100;
+  right: 0;
+  cursor: pointer;
+  border: none;
+  color: #fff;
+  font-size: 30px;
+  background-color: #6070ff;
+}
+
+.hamb {
+  color: blue;
+  margin-top: 10px;
+}
+
+.nav-tag {
+  font-family: Poppins, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 44px;
+  margin-top: 20px;
+}
+
+.menuMenu {
+  list-style: none;
+}
+
+.menuMenu a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.menuMenu:hover {
+  opacity: 2;
+}
+
+.closeIcon {
+  position: fixed;
+  z-index: 100;
+}
+
 /* Responsive design starts here */
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.